### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by HLH13297997663** [filipradenovic/cnnimageretrieval-pytorch#46]
Hello,I have a question about the loss. When I use the gem and rmac pooling in your code, the loss  decreases slow, but I use AdaptiveAvgPool method in torchvision, loss decreases fast. Does this affect retrieval accuracy? I look forward to your reply. Thank you.

**Comment by filipradenovic** [comment]
It will likely affect the retrieval accuracy, but not necessarily improve it. The loss used to train the network does not directly correspond to the average precision metric used to evaluate performance, hence, it is hard to say how the training will translate to test performance. The only way to know for sure is to evaluate the network after every (or every few) epoch on Oxford and Paris datasets.
